File: restful-server/src/main/java/via/sep/restful_server/notification/dto/NotificationDTO.java
package via.sep.restful_server.notification.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
public class NotificationDTO {
    private String type;
    private String action;
    private String entityId;
    private LocalDateTime timestamp;
    private Object data;
}

File: Blazor/RealEstateBlazor/Data/DTOs/NotificationDTO.cs
namespace RealEstateBlazor.Data.DTOs;

public class NotificationDTO
{
    public string Type { get; set; }
    public string Action { get; set; }
    public string Id { get; set; }
    public DateTime Timestamp { get; set; }
    public object Data { get; set; }
}
